### PR TITLE
macOS | Fix | Error message hidden on submit action

### PR DESCRIPTION
**Pull request by urmitchauhan**
- changes in ACRChoiceButton
- delegate method changes in Input.Toggle
- Input handler change in Input.Text
- add unit test
- ACRTextView regression line changes in ACL

**Before output** | **Changes output**
:-------------------------:|:-------------------------:
before-error-message-acl | after-error-message-acl

## Related Issue
Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
the issues fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one if reasonable to aid 
in issue tracking.

**NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
page above.

## Description
Create a [descriptive title and provide all of the relevant details in the description](https://chris.beams.io/posts/git-commit/). 
This information lives with the code and will help future readers (***including yourself***) and will serve as documentation.

At the very least please describe the issue you are addressing, and what your fix entails. 

## Sample Card
If appropriate, please include a link to a card in one of the samples directories that can be used to validate this change. This can be an existing card or a card added with this PR.

## How Verified
How you verified the fix, including one or all of the following: 
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it 
will aid in code reviews or corresponding fixes on other platforms for eg.***

### CODE REVIEW

File: source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift [hunk 1]
```diff
@@ -180,6 +180,10 @@ class ACRChoiceButton: NSView, NSTextFieldDelegate, InputHandlingViewProtocol {
         errorDelegate?.inputHandlingViewShouldShowError(self)
     }
     
+    func setShowError(_ value: Bool) {
```
Comment (mukuagar) [inline source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift:183]: use of bool value if we always pass true?

File: source/macos/AdaptiveCards/AdaptiveCards/Views/ACRSingleLineInputTextView.swift [hunk 2]
```diff
@@ -175,17 +176,9 @@ extension ACRSingleLineInputTextView: InputHandlingViewProtocol {
...
-            return self.textView.errorDelegate
-        }
-        set {
-            self.textView.errorDelegate = newValue
-        }
-    }
-    
     func showError() {
         self.textView.showError()
+        errorDelegate?.inputHandlingViewShouldShowError(self)
```
Comment (mukuagar) [inline source/macos/AdaptiveCards/AdaptiveCards/Views/ACRSingleLineInputTextView.swift:181]: you will need to manage hide error as well

**Review by mukuagar — COMMENTED** [review]
https://confluence-eng-gpk2.cisco.com/conf/display/BTC/Action+Toggle+Visibility+Test+JSONS
Please test changes against above btp
looks like there are regressions

File: source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift (hunk 1 above)
Comment (urmitchauhan) [reply source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift:183]: Yes, I had thought that. but If we will need to use it once further in a situation where its value can be both then meaning not met. believe, we ought to make it a set property.

Comment (mukuagar) [reply source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift:183]: makes sense!

File: source/macos/AdaptiveCards/AdaptiveCards/Views/ACRSingleLineInputTextView.swift (hunk 2 above)
Comment (urmitchauhan) [reply source/macos/AdaptiveCards/AdaptiveCards/Views/ACRSingleLineInputTextView.swift:189]: Hide error will handle by ACRView.

**Review by mukuagar — APPROVED** [review]

**Pull request merged**